src/components/ActionBar.vue

```diff
@@ -27,7 +27,9 @@ export default {
       isFilterMenu320Open: false,
       typeGroup: [],
       typeOptions: [
+        { label: "Election", value: "election" },
```

Comment:
We have had them removed on purpose here  #77

Reply:
ok. i fixed this in this commit 7307b8736b3bdc12591e85f35dc69881886193f2